_dashboards/get-started/time-filter.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Setting the time filter
+parent: Understanding the core concepts of OpenSearch Dashboards
+nav_order: 10
+---
+
+# Setting the time filter
+
+You can change the time range to display dashboard data over minutes, hours, days, weeks, months, or years.
+
+The default time range is **Last 15 minutes**. You can change the time range at the dashboard level or under **Stack Management > Advanced Settings > Time filter defaults**. 
+{: .note}
+
+To change the time range at the dashboard level, perform the following steps:
+
+1. Select the calendar icon.
+2. Select one of the time filter options, as shown in the following image:
+   - **Quick select:** Choose a time based on the last or next number of seconds, minutes, hours, days, or another time unit. 
+   - **Commonly used:** Choose a common time range like **Today**, **Last 7 days**, or **Last 30 days**. 
+   - **Recently used date ranges:** Select a previously used time range.
+   - **Refresh every:** Set an automatic refresh period.
+
+    <img src="{{site.url}}{{site.baseurl}}/images/dashboards/time-range.png" alt="Time range interface" width="400"/>
+
+3. Choose **Show dates** to set start and end times, and then select inside the toolbar to access the time filter pop-up window, as shown in the following image.
```

Comment:
What do we mean by "select inside the toolbar"?

Reply:
The user has to click inside the toolbar's white space to activate the popup. Do we have a better way to convey that action?

Reply:
![Screen Shot 2023-02-01 at 1 34 39 PM](https://user-images.githubusercontent.com/105296784/216157483-3e906875-e94c-4497-8cab-0a4d7046a758.png)

Reply:
![Screen Shot 2023-02-01 at 1 34 39 PM](https://user-images.githubusercontent.com/105296784/216157920-cdc0e335-5fac-4a01-ac3d-99e645d6fee1.png)

Reply:
That works. Change made.